Mathlib/Probability/Kernel/Condexp.lean

```diff
@@ -117,12 +117,7 @@ section Measurability
 variable [NormedAddCommGroup F] {f : Ω → F}
 
 theorem measurable_condExpKernel {s : Set Ω} (hs : MeasurableSet s) :
-    Measurable[m] fun ω => condExpKernel μ m ω s := by
-  nontriviality Ω
-  simp_rw [condExpKernel_apply_eq_condDistrib]
-  refine Measurable.mono ?_ (inf_le_left : m ⊓ mΩ ≤ m) le_rfl
-  convert measurable_condDistrib (μ := μ) hs
-  rw [MeasurableSpace.comap_id]
+    Measurable[m] fun ω => condExpKernel μ m ω s := Kernel.measurable_coe (condExpKernel μ m) hs
```

Comment:
```suggestion
    Measurable[m] fun ω => condExpKernel μ m ω s :=
  (condExpKernel μ m).measurable_coe hs
```
I prefer to separate clearly the statement from the proof (unless the proof is a bare `rfl`). Also, dot notation is nicer when usable (untested!)